.github/workflows/publish.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8 ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Patch for Python 3.6
+      if: matrix.python-version == 3.6
+      run: |
+        pip install strip-hints==0.1.9
+        ./patch-python3.6.sh
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements.txt
+
+    - name: Install dev dependencies
+      run: |
+        pip install -r dev_requirements.txt
+
+    - name: Test
+      run: |
+        pytest
+
+    - name: Build package
+      env:
+        PYTHON_VERSION: ${{ matrix.python-version }}
+      run: |
+        python setup.py bdist_wheel --python-tag py${PYTHON_VERSION//.}
+
+    - name: Test install package
+      run: |
+        mkdir test_install
+        cd test_install
+
+        pip install virtualenv
+        virtualenv installable
+        source installable/bin/activate
+
+        pip install $(ls "${GITHUB_WORKSPACE}/dist/*.whl")
```

Comment:
Seems to be incorrect. Same as below

Reply:
I'm not sure what is happening, so moving back to find :)